Source/DatabaseManager/DatabaseManager/Appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "SERVER = den1.mssql8.gear.host; DATABASE = theludogame; UID = theludogame; PWD = Td7p_l8gB~5T"
+  }
+}
```

Comment:
Hide this from the start 
git rm --cached <file>

.gitignore add line att bottom: 

Path/To/*.json